io/net/htx_nic_devices.py

```diff
@@ -96,7 +96,9 @@ def setUp(self):
         htx_path = os.path.join(self.teststmpdir, "HTX-master")
         os.chdir(htx_path)
 
-        exercisers = ["hxecapi_afu_dir", "hxedapl", "hxecapi", "hxeocapi"]
+        exercisers = ["hxecapi_afu_dir", "hxecapi", "hxeocapi"]
```

Comment:
can this also be ymal as input

Reply:
@PraveenPenguin ideally yes. But I do not want to dump too much into yaml and confuse the user.

And when taken hxedapl into consideration, the best approach would be to handle it in script rather than via yaml.

Consider this: I(user) do not know in which system the test is running, and so do not know whether hxedapl is needed or not on that system. Script should find it rather than me changing yaml on each system.

Reply:
@narasimhan-v sure make sense but i am thinking if we can extend support to ymal as well that help user to run customised exerciser it just add on, default flow will be same but ok with me if you don't think to have it  :)

Reply:
@PraveenPenguin I understand your point. Just that I do not think that customisation is necessary now. Once that is needed, we will definitely have a yaml parameter to take care

Reply:
@narasimhan-v sure , will merge

Reply:
@PraveenPenguin thanks :)